### PR TITLE
email-common refactoring

### PR DESCRIPTION
**Pull request by glitsj16**
Started work for https://github.com/netblue30/firejail/pull/5305. For now only balsa.profile is fully refactored for email-common.profile. claws-mail and sylpheed already were so those needed just some reshuffling.

### CODE REVIEW

File: etc/profile-a-l/balsa.profile [hunk 1]
```diff
@@ -7,76 +7,19 @@ include balsa.local
...
-nou2f
-novideo
-protocol unix,inet,inet6
-seccomp
-tracelog
-
-# disable-mnt
-# Add "pinentry-curses,pinentry-emacs,pinentry-fltk,pinentry-gnome3,pinentry-gtk,pinentry-gtk2,pinentry-gtk-2,pinentry-qt,pinentry-qt4,pinentry-tty,pinentry-x2go,pinentry-kwallet" for gpg
-# Add "ignore private-bin" for hyperlinks or have a look at the private-bins in firefox.profile and firefox-common.profile.
 private-bin balsa,balsa-ab,gpg,gpg-agent,gpg2,gpgsm
```
Comment (kmk3) [inline etc/profile-a-l/balsa.profile:20]: Why remove the comments?

Comment (glitsj16) [reply etc/profile-a-l/balsa.profile]: Oh nice catch. I only wanted to remove the `hyperlinks` comment, not the `gpg` one. Re-added, thanks!

PS: Removing the hyperlinks comment relates to the [discussion on dropping 'level 1 hacks'](https://github.com/netblue30/firejail/discussions/5566#discussioncomment-4595302) in favour of using @rusty-snake's [fireurl](https://github.com/rusty-snake/fireurl). Eager to have your input on that!

File: etc/profile-a-l/email-common.profile [hunk 2]
```diff
@@ -69,16 +73,20 @@ private-etc alternatives,ca-certificates,crypto-policies,dconf,fonts,gcrypt,gnup
...
 
+dbus-user filter
+dbus-user.talk ca.desrt.dconf
+dbus-user.talk org.freedesktop.Notifications
+dbus-user.talk org.freedesktop.secrets
+dbus-user.talk org.gnome.keyring
+dbus-user.talk org.gnome.keyring.PrivatePrompter
+dbus-user.talk org.gnome.keyring.SystemPrompter
+dbus-user.talk org.gnome.seahorse
+dbus-user.talk org.gnome.seahorse.Application
```
Comment (rusty-snake) [inline etc/profile-a-l/email-common.profile:86]:
```suggestion
dbus-user.talk org.gnome.seahorse.*
```

File: etc/profile-a-l/email-common.profile [hunk 3]
```diff
@@ -69,16 +73,20 @@ private-etc alternatives,ca-certificates,crypto-policies,dconf,fonts,gcrypt,gnup
...
 writable-run-user
+writable-var
 
+dbus-user filter
+dbus-user.talk ca.desrt.dconf
+dbus-user.talk org.freedesktop.Notifications
+dbus-user.talk org.freedesktop.secrets
+dbus-user.talk org.gnome.keyring
+dbus-user.talk org.gnome.keyring.PrivatePrompter
+dbus-user.talk org.gnome.keyring.SystemPrompter
```
[the remark this answers is not in the document]
Comment (glitsj16) [reply etc/profile-a-l/email-common.profile]: That's more elegant, thanks!

File: etc/profile-a-l/email-common.profile (hunk 2 above)
Comment (glitsj16) [reply etc/profile-a-l/email-common.profile]: Done!

**Comment by netblue30** [comment]
merged!

**Pull request merged**